torch/csrc/distributed/autograd/engine/dist_engine.cpp

```diff
@@ -131,13 +139,54 @@ void DistEngine::computeDependencies(
             //  its ancestors need to be executed as well.
             if (dynamic_cast<RecvRpcBackward*>(nextFn)) {
               recvBackwardEdges.emplace_back(edge);
+            } else if (
+                auto accumulateGradFn = dynamic_cast<AccumulateGrad*>(nextFn)) {
+              TORCH_INTERNAL_ASSERT(
+                  !accumulateGradFn->variable.defined() ||
```

Comment:
Curious, when would the variable be not defined?

Reply:
Deleted this. That may be caused by my incorrect graph modification.